lib/moon/assets/age_limit.ex

```diff
@@ -6,13 +6,16 @@ defmodule Moon.Assets.AgeLimit do
   import Moon.Helpers.Assets, only: [asset_name_to_filename: 1]
 
   prop(name, :string)
+  prop(click, :event)
+  prop(class, :string)
+
+  # All the other props below are deprecated!
+  # Please use only tailwind classes and the class prop
   prop(color, :string, values: Moon.colors())
```

Comment:
Maybe let's change these from :string to :css_class and do failover, it can be either Tailwind class name or it is made to class:

`class={@class, @color, "text-#{@color}", @bg_color, "bg-#{@bg_color}"}`